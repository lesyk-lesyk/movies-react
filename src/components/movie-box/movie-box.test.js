import React from 'react';
import { MovieBox } from './index';
import renderer from 'react-test-renderer';

describe('MovieBox', () => {
  it('renders', () => {
    const movie = {
      id: '1',
      poster_path: 'https://image.tmdb.org/t/p/w500/6o0UWX2naW7HK45PDNYmoMIk5qs.jpg',
      title: 'No Country for Old Men',
      release_date: '2007',
      genres: ['Crime', 'Drama', 'Thriller']
    };

    const tree = renderer
      .create(<MovieBox movie={movie} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
