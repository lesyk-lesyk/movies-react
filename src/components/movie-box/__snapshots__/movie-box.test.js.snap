// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieBox renders 1`] = `
<div
  className="movie-box"
>
  <img
    className="movie-image"
    src="https://image.tmdb.org/t/p/w500/6o0UWX2naW7HK45PDNYmoMIk5qs.jpg"
  />
  <div
    className="movie-info"
  >
    <div
      className="title"
    >
      No Country for Old Men
    </div>
    <div
      className="year"
    >
      2007
    </div>
    <div
      className="genres"
    >
      <span>
        Crime
      </span>
      <span>
        Drama
      </span>
      <span>
        Thriller
      </span>
    </div>
  </div>
</div>
`;
